test/integration/suites/ghostunnel-federation/conf/downstream/server/server.conf

```diff
@@ -12,7 +12,7 @@ server {
         bundle_endpoint {
             port = 8443
         }
-        federates_with "spiffe://upstream-domain.test" {
+        federates_with "upstream-domain.test" {
```

Comment:
Should we keep some integration tests that include the `spiffe://` prefix to ensure it's still working and there haven't been any backwards-incompatible changes?

Reply:
Makes sense to me. I will revert this change so we have one server configured using a trust domain name and the other using a trust domain SPIFFE ID. Also, will add some comments to make this point clear.